### PR TITLE
DOCS-261 - Rework cloud soar

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request makes significant changes to the Cloud SOAR documentation to bring it up-to-date with the latest version of Cloud SOAR (Delivery 2). These articles are the main ones updated:
* https://help.sumologic.com/docs/cloud-soar/overview/
* https://help.sumologic.com/docs/cloud-soar/incidents-triage/

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-261](https://sumologic.atlassian.net/browse/DOCS-261)

### CODE REVIEW

File: docs/cloud-soar/legacy/legacy-cloud-soar-automation.md [hunk 1]
```diff
@@ -12,11 +12,73 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+You can use CyberArk Credential Manager to manage data that will be used in integration resources.
+
+<img src={useBaseUrl('img/cloud-soar/cyberArk1.png')} alt="Integrations" style={{border: '1px solid gray'}} width="600"/>
+
+Using the cogwheel icon on the right in the integrations section, the main section of the CyberArk configuration opens.
+
+<img src={useBaseUrl('img/cloud-soar/CyberArk2.png')} alt="CyberArk configuration" style={{border: '1px solid gray'}} width="400"/>
+
+Here you can set URL and port of the Components server, and the credentials needed to connect to CyberArk. The Enable checkbox can be enabled or disabled later.
```
Comment (kimsauce) [inline docs/cloud-soar/legacy/legacy-cloud-soar-automation.md:29]:
```suggestion
Here you can set URL and port of the Components server, and the credentials needed to connect to CyberArk. The **Enable** checkbox can be enabled or disabled later.
```

File: docs/cloud-soar/legacy/legacy-cloud-soar-automation.md [hunk 2]
```diff
@@ -12,11 +12,73 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+If enabled, when you go to open the detail of a integration resource you'll find a new checkbox (**Use CyberArk fields**) at the top already active. If the checkbox on above window is disabled, the checkbox in the resource window will be disabled by default, and it will not be possible to activate it.
+
+<img src={useBaseUrl('img/cloud-soar/CyberArk3.png')} alt="Enable CyberArk fields" style={{border: '1px solid gray'}} width="800"/>
+
+If the checkbox **Use CyberArk fields** is enabled, two new mandatory fields will appear:
+* **Account Name** > userName in CyberArk
+* **Platform ID** > platformId in CyberArk
+
+Near to the fields there will be the relative toggle that will enable the related field for use on CyberArk.
```
Comment (kimsauce) [inline docs/cloud-soar/legacy/legacy-cloud-soar-automation.md:39]:
```suggestion
Near to the fields, there will be the relative toggle that will enable the related field for use on CyberArk.
```

File: docs/cloud-soar/legacy/legacy-cloud-soar-automation.md [hunk 3]
```diff
@@ -12,11 +12,73 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+<img src={useBaseUrl('img/cloud-soar/CyberArk5.png')} alt="CyberArk fields enabled" style={{border: '1px solid gray'}} width="400"/>
+
+In the image above, you can see two custom fields of the resource with their toggles. The first field has been enabled to use CyberArk, while the second not.
```
Comment (kimsauce) [inline docs/cloud-soar/legacy/legacy-cloud-soar-automation.md:43]:
```suggestion
In the image above, you can see two custom fields of the resource with their toggles. The first field has been enabled to use CyberArk, while the second one hasn't.
```

**Review by kimsauce — APPROVED** [review]
this looks great. one possible edit.

**Comment by jpipkin1** [comment]
Wow! Thanks for the review, @kimsauce!

**Pull request merged**